File: dotNet/MVC/MvcApp/MvcApp/Models/Services/LoginService.cs
using MvcApp.Models.Login;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace MvcApp.Models.Services
{
    public class LoginService
    {
        ContactDbContext dbContext;
        public LoginService()
        {
            dbContext = new ContactDbContext();
        }

        public bool AuthenticateUser(string username, string password)
        {
            User user = dbContext.Users.Where((u) => u.Name == username && u.Password == password)
                 .FirstOrDefault();
            if (user != null)
            {
                return true;
            }
            return false;
        }

    }
}

File: Angular/contact-app/src/app/services/LoginService.ts

import {IUser} from "./IUser";
export class LoginService{

    users:IUser[];
    loggedInUser:string;
    constructor(){
       this.users=[{id:"dharmesh",password:"111"},{id:"chandan",password:"222"}];      
    }

    authenticateUser(inputId:string,inputPass:string):boolean{
      for(let user of this.users){
        if(user.id==inputId && user.password==inputPass){
            this.loggedInUser=inputId;
            return true;
        }
        
      }
      return false;
       
    }

}